crates/client/db/src/db_metrics.rs

```diff
@@ -1,15 +1,34 @@
-use mc_metrics::{IntGaugeVec, MetricsRegistry, Opts, PrometheusError};
+use crate::{Column, DatabaseExt, DB};
+use mc_metrics::{Gauge, IntGaugeVec, MetricsRegistry, Opts, PrometheusError, F64};
 
 #[derive(Clone, Debug)]
 pub struct DbMetrics {
+    pub db_size: Gauge<F64>,
     pub column_sizes: IntGaugeVec,
 }
 
 impl DbMetrics {
     pub fn register(registry: &MetricsRegistry) -> Result<Self, PrometheusError> {
         Ok(Self {
+            db_size: registry.register(Gauge::new("db_size", "Node storage usage in GB")?)?,
             column_sizes: registry
                 .register(IntGaugeVec::new(Opts::new("column_sizes", "Sizes of RocksDB columns"), &["column"])?)?,
         })
     }
+
+    /// Returns the total storage size
```

Comment:
why would you want it behind a compilation flag?

yep it's once every 200 blocks and I think the way it's handled is bad - it should instead be like the maybe db flush thing -- at the end of every block, check that 5s(?) has elapsed since the last metrics update and if so, update
it would make much more sense to handle it that way so that it continues to work fine when the chain is fully synced and new blocks don't come that often

but also, it's kind of inexpensive i believe - it's stats that rocksdb already tracks, and updating the prometheus metrics lowers to nothing but a few atomic stores under the hood so i'm also inclined to not bother and put it every block.

Reply:
i changed it to the 5s thinggy